src/main/kotlin/org/jitsi/jibri/selenium/pageobjects/CallPage.kt

```diff
@@ -35,38 +37,43 @@ class CallPage(driver: RemoteWebDriver) : AbstractPageObject(driver) {
         PageFactory.initElements(driver, this)
     }
 
+    @ExperimentalTime
     override fun visit(url: String): Boolean {
-        logger.debug { "Visiting url $url" }
         if (!super.visit(url)) {
             return false
         }
-        val start = System.currentTimeMillis()
-        return try {
-            WebDriverWait(driver, 30).until {
-                val result = driver.executeScript(
-                    """
+        val (result, totalTime) = measureTimedValue {
+            try {
+                WebDriverWait(driver, 30).until {
+                    val result = driver.executeScript(
+                        """
                     try {
```

Comment:
nit: i think this block should be indented once more.

Reply:
Done